arch/arm/boot/dts/bcm2836-rpi-2-b.dts

```diff
@@ -10,7 +10,8 @@
 	compatible = "raspberrypi,2-model-b", "brcm,bcm2836";
 	model = "Raspberry Pi 2 Model B";
 
-	memory {
+	memory@0 {
```

Comment:
This will need a minor change to the firmware (it's on the list).

Reply:
Oops, i assumed the memory node handle is independent of the model.

Reply:
It is common to all models, I just this instance as an example.

Reply:
I dropped the relevant changes in this PR, but they are still present in upstream.

The consequences could be a wrong memory size?

Reply:
Correct. Platforms with multiple memory size variants rely on the firmware to correct the size. I have a small patch to the firmware that should handle the change, so if you restore the "memory@0" change I can test it.

Reply:
I've pushed two patches to switch all DTS files to "memory@0".